### DISCUSSION TITLE
Pimcore Academy: Document properties

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#9465] [Q&A / Support]
> Question 7: Which statements concerning properties are correct?

Alternative *Properties can be used to influence navigation building.* is marked as wrong even though properties are used in navigation building, for example: `$page->setProperty('navigation_name')` and `$page->setProperty('navigation_exclude')`:

https://github.com/pimcore/pimcore/blob/4550a053fac47a1d2c0406bf15c9799ef078bbb7/lib/Navigation/Builder.php#L319-L336

**Comment by fashxp** [comment]
hmm ... should be marked as true, also just verified. so maybe please try once more.
